bin/solana-sys-tuner.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Solana System Tuner
+After=network.target
+Before=sol.service
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=sol
```

Comment:
User needs to be `root` (which is a little 😬 since it's running out of `/home/sol/bin` that's writable by the sol user.   I think this is tolerable for our setup but really bad practice in general)